src/Resources/views/AmazonPay/amazonPayStart.html.twig

```diff
@@ -1,5 +1,3 @@
-{% include '@SyliusUi/_javascripts.html.twig' with {'path': 'assets/shop/js/app.js'} %}
```

Comment:
these lines should not be removed, but replaced with:

```twig
{{ encore_entry_script_tags('shop-entry', null, 'shop') }}
```

Reply:
But we didn't need the shop javascript here. all the code is plain javascript here without dependencies.
But i can readd it if we need it.